src/Components/Error/Error.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import './Error.scss'
+import gif from '../../images/giphy.gif'
+
+const Error = ({errorStatus}) => {
+
+  if (errorStatus >= 400 && errorStatus <= 499) {
+    return (
+      <main className='main'>
+        <img className='catGif' src={gif}/>
+      </main>
+    )
+
+  } else if (errorStatus >= 499) {
+    return (
+      <main className='main'>
+        <h1 className='errorStatus'>500 Error</h1>
+        <h2 className='errorMsg'>Oops! Something went wrong. Please try refreshing the page.</h2>
+      </main>
+    )
+  }
+}
+
+export default Error
```

Comment:
Very nice! I like our 400 UI! Just to be picky we could say > 499 instead of >= 499.